tests/Wallabag/CoreBundle/Controller/TagControllerTest.php

```diff
@@ -126,9 +126,13 @@ public function testRemoveTagFromEntry()
             ->getRepository('WallabagCoreBundle:Tag')
             ->findOneByEntryAndTagLabel($entry, $this->tagName);
 
+        // We make a first request to set an history and test redirection after tag deletion
+        $client->request('GET', '/view/'.$entry->getId());
```

Comment:
This should be fixed.
I don't know why it worked before...
I just re-retrieved the entry to be sure to then fetch tags correctly, and it works.